flexget/plugins/sites/rutracker.py

```diff
@@ -95,15 +98,17 @@ def __init__(self, login, password, cookies=None, db_session=None):
     def __call__(self, r):
         url = r.url
         id = re.findall(r'\d+', url)[0]
```

Comment:
While you're already in there, you should change this variable name to something other than `id`.